qa/suites/rados/mgr/tasks/devicehealth.yaml

```diff
@@ -0,0 +1,15 @@
+tasks:
+  - install:
+  - ceph:
+      # tests may leave mgrs broken, so don't try and call into them
+      # to invoke e.g. pg dump during teardown.
+      wait-for-scrub: false
+      log-whitelist:
+        - overall HEALTH_
+        - \(MGR_DOWN\)
+        - \(PG_
+        - replacing it with standby
+        - No standby daemons available
+  - cephfs_test_runner:
+      modules:
+        - tasks.mgr.test_devicehealth
```

Comment:
No need for this yaml file -- new methods in an existing module will get invoked automatically (you'd only need this if creating a new test_devicehealth.py file).  We only have the separate modules for cases where the mgr module has more complex tests than just invoking the selftest method.